packages/web-service/src/pages/habitat/a24/check-habitat-answers/check-habitat-answers.js

```diff
@@ -0,0 +1,129 @@
+import Joi from 'joi'
+import pageRoute from '../../../../routes/page-route.js'
+import { habitatURIs, TASKLIST } from '../../../../uris.js'
+import { APIRequests } from '../../../../services/api-requests.js'
+import { PowerPlatformKeys } from '@defra/wls-powerapps-keys'
+import { SECTION_TASKS } from '../../../tasklist/licence-type-map.js'
+const {
+  METHOD_IDS: { OBSTRUCT_SETT_WITH_BLOCK_OR_PROOF, OBSTRUCT_SETT_WITH_GATES, DAMAGE_A_SETT, DESTROY_A_SETT, DISTURB_A_SETT },
+  SETT_TYPE: { MAIN_NO_ALTERNATIVE_SETT, ANNEXE, SUBSIDIARY, OUTLIER }
+} = PowerPlatformKeys
+
+const addSett = 'additional-sett'
+const settDisruptionMethods = [
+  {
+    value: OBSTRUCT_SETT_WITH_GATES,
+    text: 'Obstructing sett entrances by means of one-way badger gates'
```

Comment:
can you move the page text into the templates please

Reply:
The only places that display this text already have the text moved in, and this is purely for mapping on the check you answers page.